web/src/components/package/PackageDetail.vue

```diff
@@ -177,6 +177,19 @@
                   Published on: {{ packageInfo.createdAt | formatDate }}
                 </div>
               </nav>
+              <nav class="panel">
+                <p class="panel-heading">
+                  <font-awesome-icon class="icon-margin-right" :icon="['fas', 'shield-alt']" />Audit
+                </p>
+                <div class="panel-block" v-if="malicious">
+                  <font-awesome-icon class="icon-margin-right" :icon="['fa', 'biohazard']" />
+                  Malicious module: use at your own risk
+                </div>
+                <div class="panel-block" v-else>
+                  <font-awesome-icon class="icon-margin-right" :icon="['fa', 'lock']" />
+                  The module is safe
```

Comment:
I don't think we should display that it is "safe," as it requires someone reporting that the module is malicious for us to flag it as such. Instead, I think we should just remove that section all together and only warn the user if it is malicious.

Reply:
We can do that for now, however I want to display permissions, etc. there too in the future

Reply:
I see. That would be neat. We'll have to figure out a good way to organize that when the time comes.